### PR TITLE
feat(primitives): add HorizontalBarsFilter icons

### PR DESCRIPTION
**Pull request by Achintha444**
### Purpose
feat(primitives): add HorizontalBarsFilter icons

### Related Issues
- https://github.com/wso2/oxygen-ui/issues/251

### Related PRs
- N/A

### Checklist
- [ ] UX/UI review done on the final implementation.
- [ ] Story provided. (Add screenshots)
- [ ] Manual test round performed and verified.
- [ ] Unit tests provided. (Add links if there are any)
- [ ] Documentation provided. (Add links if there are any)

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: packages/primitives/src/icons/three-bars-middle-16.svg [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
...88 13.3529C6.88588 13.3529 6.71827 13.2853 6.58306 13.1499C6.44769 13.0145 6.38 12.8468 6.38 12.6468C6.38 12.4467 6.44769 12.2791 6.58306 12.144C6.71827 12.0088 6.88588 11.9412 7.08588 11.9412H8.90494C9.10494 11.9412 9.27263 12.0089 9.408 12.1442C9.54322 12.2796 9.61082 12.4473 9.61082 12.6473C9.61082 12.8475 9.54322 13.0151 9.408 13.1501C9.27263 13.2853 9.10494 13.3529 8.90494 13.3529H7.08588ZM3.43882 8.88235C3.23882 8.88235 3.07122 8.81467 2.936 8.67929C2.80078 8.54392 2.73318 8.37624 2.73318 8.17624C2.73318 7.97608 2.80078 7.80847 2.936 7.67341C3.07122 7.5382 3.23882 7.47059 3.43882 7.47059H12.552C12.752 7.47059 12.9197 7.53827 13.0551 7.67365C13.1903 7.80902 13.2579 7.97671 13.2579 8.17671C13.2579 8.37686 13.1903 8.54447 13.0551 8.67953C12.9197 8.81475 12.752 8.88235 12.552 8.88235H3.43882ZM0.705882 4.41176C0.505882 4.41176 0.338275 4.34408 0.203059 4.20871C0.0676864 4.07333 0 3.90565 0 3.70565C0 3.50549 0.0676864 3.33788 0.203059 3.20282C0.338275 3.06761 0.505882 3 0.705882 3H15.2941C15.4941 3 15.6617 3.06769 15.7969 3.20306C15.9323 3.33843 16 3.50612 16 3.70612C16 3.90627 15.9323 4.07388 15.7969 4.20894C15.6617 4.34416 15.4941 4.41176 15.2941 4.41176H0.705882Z" fill="black"/>
```
Comment (savindi7) [inline packages/primitives/src/icons/three-bars-middle-16.svg:20]: Shall we indent this?

**Comment by savindi7** [comment]
Shall we rename the icon to something like HorizontalBarsFilter or HorizontalBarsMiddle ?

**Comment by Achintha444** [comment]
> HorizontalBarsFilter

Ack, will change the name to `HorizontalBarsFilter`

File: packages/primitives/src/icons/three-bars-middle-16.svg (hunk 1 above)
Comment (Achintha444) [reply packages/primitives/src/icons/three-bars-middle-16.svg]: fixed with https://github.com/wso2/oxygen-ui/pull/252/commits/c53520ccb51918cfaf1570c714d70d9370b50f6b

**Review by pavinduLakshan — APPROVED** [review]

**Pull request merged**